packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -228,16 +228,19 @@ const DomainPickerSuggestionItem: React.FC< Props > = ( {
 				{ isFree && ! isUnavailable && freeDomainLabel }
 				{ ! isFree && ! isUnavailable && (
 					<>
-						<span className="domain-picker__price-inclusive">
-							{ /* Intentional whitespace to get the spacing around the text right */ }{ ' ' }
-							{ paidIncludedDomainLabel }{ ' ' }
-						</span>
 						<span className="domain-picker__price-cost">
 							{
 								/* translators: %s is the price with currency. Eg: $15/year. */
 								sprintf( __( '%s/year', __i18n_text_domain__ ), cost )
 							}
 						</span>
+						<span className="domain-picker__price-inclusive">
+							{ /* Intentional whitespace to get the spacing around the text right */ }{ ' ' }
+							{ paidIncludedDomainLabel }{ ' ' }
+						</span>
+						<span className="domain-picker__price-renewal">
+							{ sprintf( __( 'Renews at: %s / year', __i18n_text_domain__ ), cost ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Renews at: %s /year' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/focused-launch/domain-picker-renewal-price-label)